Kernel/Time/TimeManagement.cpp

```diff
@@ -474,6 +481,7 @@ UNMAP_AFTER_INIT bool TimeManagement::probe_and_set_aarch64_hardware_timers()
 
     return true;
 }
+#elif ARCH(RISCV64)
```

Comment:
These empty elifs could probably do with some empty functions with TODO_RISCV in them so that we don't forget to impl them when the time comes (pun intended).